services/account/scripts/docker-clean-up.sh

```diff
@@ -2,12 +2,13 @@
 
 # Stop and remove containers, networks
 echo "Stopping and removing containers, networks..."
-docker compose down
+docker compose down frequency redis account-service-api account-service-worker
 
 # Remove specified volumes
 echo "Removing specified volumes..."
-docker volume rm account-service_redis_data
-docker volume rm account-service_chainstorage
+docker volume rm gateway_redis_data
```

Comment:
docker-compose.yaml file moved to root, so targets needed to be renamed to match.